monai/metrics/rocauc.py

```diff
@@ -81,7 +81,7 @@ def _calculate(y_pred: torch.Tensor, y: torch.Tensor) -> float:
         return float("nan")
 
     n = len(y)
-    indices = y_pred.argsort()
+    indices = y_pred.argsort().cpu()
     y = y[indices].cpu().numpy()
```

Comment:
I think the problem is that why `y` and `y_pred` are not on the same device in the latest tutorial?
If both are on GPU, `y[indices]` may be faster than CPU indices? So I think we should change the tutorial instead of here?

Thanks.

Reply:
sure, I revert the change here